core/src/storage/__init__.py

```diff
@@ -20,6 +16,14 @@ def init_unlocked() -> None:
     version = device.get_version()
     if version == common.STORAGE_VERSION_01:
         _migrate_from_version_01()
+    if device.is_version_stored() and not device.is_initialized():
```

Comment:
A comment here would be helpful and whitespace to separate this block from "# Check for storage version upgrade".

Reply:
Thanks, fixed in afd8acb25f31dc8d8cedc15576f3b37262f7a455.